src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,36 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  currentTodo: Todo;
+  onCloseModal: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  currentTodo,
+  onCloseModal: onClose,
+}) => {
+  const {
+    id,
+    title,
+    completed,
+    userId,
+  } = currentTodo;
+
+  const [user, setUser] = useState<User>();
```

Comment:
In most cases, we use null if the user does not exist, I'm not sure what type of User is gonna be here